test/config/featureSwitches/FeatureSwitchSpec.scala

```diff
@@ -49,30 +49,12 @@ class FeatureSwitchSpec extends SpecBase {
   }
 
   "FeatureSwitching isEnabled" should {
-    s"return true if ETMP feature switch is enabled" in new Setup {
-      featureSwitching.enableFeatureSwitch(CallETMP)
-      featureSwitching.isEnabled(CallETMP) shouldBe true
-    }
-
-    s"return.false if ETMP feature switch is disabled" in new Setup {
-      featureSwitching.disableFeatureSwitch(CallETMP)
-      featureSwitching.isEnabled(CallETMP) shouldBe false
-    }
-
-    "return false if ETMP feature switch does not exist" in new Setup {
-      featureSwitching.isEnabled(CallETMP) shouldBe false
-    }
 
     s"return true if PEGA feature switch is enabled" in new Setup {
       featureSwitching.enableFeatureSwitch(CallPEGA)
       featureSwitching.isEnabled(CallPEGA) shouldBe true
     }
 
-    s"return.false if PEGA feature switch is disabled" in new Setup {
```

Comment:
This test needs to be added back in

Reply:
Yes, also I'm adding back tests for CallAPI1812ETMP and CallAPI1811ETMP